app/assets/javascripts/sumoselect.js.coffee

```diff
@@ -0,0 +1,4 @@
+$(document).ready ->
+  $('.dept_select').SumoSelect(
+    placeholder: 'Select department')
+  return
```

Comment:
One line would be fine, and it'd be good if it indicated that one can select multiple departments:

```coffee
$(document).ready ->
  $('.dept_select').SumoSelect placeholder: 'Select department(s)'
  return
```